src/main/java/sh/damon/stackmob/config/EntityConfig.java

```diff
@@ -0,0 +1,14 @@
+package sh.damon.stackmob.config;
+
+import net.minecraft.util.Identifier;
+
+public class EntityConfig {
+    public Identifier entityId = Identifier.ofVanilla("sheep");
```

Comment:
Im guessing that this is going to change to a more generic Identifier of some sorts? Now it is sheep specific

Reply:
Looks like a default identifier. Its okay then